apollo-openapi/src/test/java/com/ctrip/framework/apollo/openapi/client/ApolloOpenApiClientTest.java

```diff
@@ -32,6 +34,71 @@ public void testCreate() {
     assertEquals(someUrl, client.getPortalUrl());
     assertEquals(someToken, client.getToken());
   }
+  /**
+   * please config correct value in testCRUD,then you can remove exception
+   */
+  @Test(expected = Exception.class)
+  public void testIllegalKey(){
+    testCRUD("/");
+    testCRUD(".");
+    testCRUD("a.b/c");
+    testCRUD("<");
+    testCRUD(">");
+    testCRUD("\\");
+    testCRUD("`");
+    testCRUD("{");
+    testCRUD("}");
+    testCRUD("[");
+    testCRUD("]");
+    testCRUD("\"");
+    testCRUD("'");
+  }
+
+  public void testCRUD(String key) {
+    String someUrl = "http://localhost:8070";
+    String someToken = "someToken";
+    String appId = "test1";
+    String env = "DEV";
+    String cluster = "default";
+    String namespace = "application";
+    String value = "abc";
+    String updateValue = "ddd";
+    String operator = "apollo";
+
+    ApolloOpenApiClient client = ApolloOpenApiClient.newBuilder().withPortalUrl(someUrl).withToken(someToken).build();
+
+    OpenItemDTO createItem = new OpenItemDTO();
+    createItem.setKey(key);
+    createItem.setValue(value);
+
+    createItem.setDataChangeCreatedBy(operator);
+    OpenItemDTO item1 = client.createItem(appId, env, cluster, namespace, createItem);
+    System.out.println("create item : "+item1);
```

Comment:
It's not a good practice to use the System.out in unit test as it's not testable.
Instead, we could use mockito and junit to verify the logic.
BTW, it's better to verify the item logic in `ItemOpenApiServiceTest`.

Reply:
OK， System.out is just for output, I'll remove them and  transfer the item vferify logic in ItemOpenApiServiceTest.